web/viewer.css

```diff
@@ -291,7 +295,7 @@ html[dir='rtl'] #toolbarSidebar {
 
 #toolbarContainer, .findbar, .secondaryToolbar {
   position: relative;
-  height: 32px;
+  height: auto;
```

Comment:
But you've already added that at https://github.com/mozilla/pdf.js/pull/6274/files#diff-5a36ed3216dce1e4da7e938ee4503ebdR379, which definitely should be enough!
This line, however, *needs* to be removed, since it breaks things (as explained above):
![overlap](https://cloud.githubusercontent.com/assets/2692120/8935947/c7f2aab6-354f-11e5-9485-c1ce454037b3.png)